webui/react/src/e2e/tests/experimentList.spec.ts

```diff
@@ -19,11 +19,11 @@ test.describe('Experiment List', () => {
   test.beforeAll(async ({ backgroundAuthedPage, newWorkspace, newProject }) => {
     const projectDetailsPageSetup = new ProjectDetails(backgroundAuthedPage);
     await projectDetailsPageSetup.gotoProject(newProject.response.project.id);
-    await test.step('Create an experiment', async () => {
+    await test.step('Create experiments', async () => {
       await expect(
         projectDetailsPageSetup.f_experimentList.tableActionBar.count.pwLocator,
       ).toContainText('experiment');
-      Array(4)
+      Array(52)
```

Comment:
the count here is increased to ensure the page size options show up

Reply:
if this isn't necessary for the other tests, i would ask that pagination tests get their own describe block for the other experiments

Reply:
given that the project will be shared between the describe blocks (the project fixture has to be scoped to the worker), i think separating these blocks will only serve to make this test suite take longer.

is that not true?

Reply:
if it's shared then it's no problem

Reply:
you could nest a new describe block inside this one with 48 more. that way if I individually run one of the other tests, I won't get 50 experiments that I don't need

Reply:
you could nest a new describe block inside this one with 48 more. that way if I individually run one of the other tests, I won't get 50 experiments that I don't need

Reply:
Good idea -- i implemented it.